plugins/calendar/service.js

```diff
@@ -148,27 +151,17 @@
 							recuring_event.startName = startDate.calendar().toUpperCase();
 							recuring_event.end = endDate;
 							recuring_event.endName = endDate.subtract(1, 'seconds').calendar().toUpperCase();
-
-							if (recuring_event.startName && recuring_event.endName) {
-								if (recuring_event.startName == recuring_event.endName) {
-									recuring_event.label = recuring_event.startName
```

Comment:
hm.. I did a diff of my earlier pre-plugin implementation, and found no changes there..  
so that should not have been removed. 

all this change does is remove the hard coded data storage name (service.events) from all the routines and have them take a passed in storage area for the array of events.